src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { CSSProperties } from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { ColoredTd } from './ColoredTd';
+
+type SnazzyButtonProps = {
+  color: string;
+  isSnazzy: boolean;
+  customStyle?: string;
+} & ButtonProps;
+
+const SnazzyButton: React.FC<SnazzyButtonProps> = ({ color, isSnazzy, customStyle, ...rest }) => {
+  if (isSnazzy) {
+    return (
+      <ColoredTd color={color}>
+        <Button style={customStyle as CSSProperties} {...rest} />
```

Comment:
Very cool, this is neat and should work! But do re-read the task requirements: 

_"Using "react-jss" and "createUseStyles""_

The goal of this task was to get you used to using createUseStyles/react-jss.